src/cloudant/database.py

```diff
@@ -1098,3 +1101,124 @@ def get_query_result(self, selector, fields=None, raw_result=False,
             return QueryResult(query, **kwargs)
         else:
             return query.result
+
+    def get_search_result(self, ddoc_id, index_name, query, **kwargs):
+        """
+        Retrieves the raw JSON content from the remote database based on the
+        search index on the server, using the kwargs provided as query
+        parameters.
+
+        Example for search queries:
+
+        .. code-block:: python
+
+            # Assuming that 'searchindex001' exists as part of the
+            # design document ddoc in the remote database...
+            # Retrieve documents where the Lucene field name is 'name' and
+            # the value is 'julia*'
+            query = 'name:julia*'
+            resp = db.get_search_result(ddoc, 'searchindex001', query,
+                                        include_docs=True)
```

Comment:
I think your example needs to be reworked slightly now that you are using ddoc_id instead of a full DesignDocument (ddoc).